db/migrate/20160719100630_create_wishlists.rb

```diff
@@ -0,0 +1,9 @@
+class CreateWishlists < ActiveRecord::Migration[5.0]
+  def change
+    create_table :wishlists do |t|
+      t.string :name, null: false, default: ""
+      t.text :description, null: false, default: ""
```

Comment:
why you set default for "" and present true, it will create you empty data for them, I will drop default, helps nothing